app/js/lib/foxbox/services/base.js

```diff
@@ -65,40 +64,46 @@ export default class BaseService extends EventDispatcher {
   }
 
   /**
-   * Call a service setter with a value.
+   * Sends a value to a service channel.
    *
-   * @param {Object|string} selector A value matching the kind of a channel.
-   * @param {string|number|boolean} value
+   * @param {Object|string} selector Selector for the channel to use.
+   * @param {*} value Value to send to the channel.
    * @return {Promise}
    */
-  set(selector, value = '') {
-    const setter = this[p.getSendChannel](selector);
-    const setterType = this[p.getSetterValueType](setter.kind);
-
-    return this[p.api].put(
-      'channels/set',
-      [[{ id: setter.id }, { [setterType]: value }]]
-    );
+  send(selector, value = null) {
```

Comment:
I'll revert it back to ```get``` and ```set``` then, since there is a rule - `if not critical, don't change if there is no 100% agreement` :)